### PR TITLE
The Ember Times No. 139 - March 13th 2020

### PR DESCRIPTION
**Pull request by amyrlam**
📝 Ideas, feel free to add to list or claim!

- [ ] RFC: `{{set}}` helper https://github.com/emberjs/rfcs/pull/594
- [ ] RFC: Test colocation https://github.com/emberjs/rfcs/pull/599 (reach out to @NullVoxPopuli if questions!)
- [x] Ember Twiddle update (🔏 @ijlee2 )
  - https://twitter.com/gaurav9576/status/1234944600177070087
  - https://twitter.com/gaurav9576/status/1238442737986351104
- [x] Ember Test Case for a CRUD Application with Mirage
  - https://medium.com/@srajas02/ember-test-case-for-a-crud-application-with-mirage-d6d9836bfee2 (🔏 @ijlee2 )
- [x] ember-models-table v3 released (🔏 @MinThaMie )
  - https://twitter.com/oonechiporenko/status/1235571293530902531
- [x] COVID map is emberjs https://twitter.com/odoenet/status/1233439467421065216 (:lock: @jaredgalanis)
- [x] Ember Chennai February Meetup videos posted (🔏 @abhilashlr)
- [ ] An update for `rwjblue-style-release-it-setup` by Robert Jackson (rwjblue)
  - https://discordapp.com/channels/480462759797063690/491905849405472769/686891414738829343
- [x] ember-jsqr release (🔏 @NullVoxPopuli )
- [x] RFC "Destroyables": https://github.com/emberjs/rfcs/pull/580 (🔏 @jessica-jordan)
- [x] EmberConf update on remote setup? (🔏 @jessica-jordan)
- [ ] Respect the Underscore https://medium.com/@ritual_steven/respect-the-underscore-331fa23e1595

### Outstanding Team Quotes about Octane from https://github.com/ember-learn/ember-blog/pull/459

- [ ] @MelSumner
- [ ] @kategengler
- [ ] @mixonic
- [ ] @pzuraq

- [ ] More octane at company x stories? Square, others?

📆 Future stories
- [ ] Check RFCs: https://github.com/emberjs/rfcs/pulls
- [ ] **UPDATE** _this should go out in the 2/28 issue per 2/20 learning team meeting_ Write up on current website update status. Call for contributors, explaining the styleguide and which projects contributors can help with. Ping @mansona when this topic is due for release in the Times (3/7 note: is this still valid?)

🤔 Questions?

See #support-ember-times on Discord or CONTRIBUTING.

✅ Todos:

- [ ] check all header links are also in the body, and add to template

- Check https://github.com/ember-learn/whats-new-in-emberland (roundup of all the PRs in emberjs and ember-learn repos) for ideas
- Use https://github.com/ember-learn/emberjs-times-tools README
- Check Netlify preview, to ensure display on emberjs.com/blog is good
  - [ ] Check buttons
- [ ] Crosspost on dev.to, delete Readers Question picture, add Canonical URL 
- [ ] Set up tweet for Monday morning on tweetdeck
  - [ ] post $natural

### CODE REVIEW

**Comment by abhilashlr** [comment]
- [x] EmberJS Chennai meet-up video playlist - I can write on this.

File: source/2020-03-13-the-ember-times-issue-139.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+<!--alex ignore host-hostess-->
+But even though these APIs are already sufficient to go about our lives as application developers, it is still a limiting way for addon authors to create useful community projects. What if your addon could add teardown work to host apps in a **non-hierarchical** manner, so it can never conflict with other addon's teardown functionalities?
+
+If you want to know the answer to this question, you shouldn't miss out on reading [Chris Garrett's (@pzuraq)](https://github.com/pzuraq) recent **Request for Comments (RFC)** about [Destroyables](https://github.com/emberjs/rfcs/blob/destroyables/text/0580-destroyables.md) - an API proposal for addon authors and framework contributors. Be sure to join the discussion in the comments section of [the original RFC](https://github.com/emberjs/rfcs/pull/580)!
+
+---
+
+## [EmberConf 2020 is on (and goes virtual) 📺🎉](https://emberconf.com/#/emberconf-update-march-12)
```
Comment (chancancode) [inline source/2020-03-13-the-ember-times-issue-139.md]: Should we promote this to the top?

Comment (amyrlam) [reply source/2020-03-13-the-ember-times-issue-139.md]: @chancancode thanks for the ping - yes

**Comment by amyrlam** [comment]
Netlify is borked, but lgtm

image

**Pull request merged**